tests/tendermint/test_core.py

```diff
@@ -1,10 +1,13 @@
 import json
 
+import pytest
```

Comment:
Instead of marking each function, you can apply the marker to the [whole module](https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules):

```python
pytestmark = pytest.mark.tendermint
```

Reply:
yes, that should be better for all tests under `tests/tendermint/`, for others we individually whitelist.